WordPress/src/main/java/org/wordpress/android/models/ReaderBlog.java

```diff
@@ -54,15 +56,22 @@ public static ReaderBlog fromJson(JSONObject json) {
             blog.isFollowing = true;
         } else {
             blog.blogId = json.optLong("ID");
+            blog.feedId = json.optLong("feed_ID");
             blog.setName(JSONUtil.getStringDecoded(json, "name"));
             blog.setDescription(JSONUtil.getStringDecoded(json, "description"));
             blog.setUrl(JSONUtil.getString(json, "URL"));
+            blog.setFeedUrl(JSONUtil.getString(json, "feed_URL"));
             blog.isJetpack = JSONUtil.getBool(json, "jetpack");
             blog.isPrivate = JSONUtil.getBool(json, "is_private");
             blog.isFollowing = JSONUtil.getBool(json, "is_following");
             blog.numSubscribers = json.optInt("subscribers_count");
         }
 
+        // blogId will be empty for feeds, so set it to the feedId (consistent with /read/ endpoints)
+        if (blog.blogId == 0 && blog.feedId != 0) {
+            blog.blogId = blog.feedId;
```

Comment:
I'm not sure how this is used later, but be very careful that these IDs are not used in the wrong context. Feed ID and Blog ID can and do overlap.